nflow-tests/src/test/java/io/nflow/tests/SkipAutoStartTest.java

```diff
@@ -0,0 +1,26 @@
+package io.nflow.tests;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import io.nflow.tests.runner.NflowServerRule;
+
+public class SkipAutoStartTest extends AbstractNflowTest {
+
+  @ClassRule
+  public static NflowServerRule server = new NflowServerRule.Builder()
```

Comment:
Having a non-database connection didn't work, as there is code that establishes the connection to the database here:

https://github.com/NitorCreations/nflow/blob/master/nflow-engine/src/main/java/io/nflow/engine/internal/storage/db/DatabaseConfiguration.java#L36

That does (and IMO should) fail when initializing NFlow. But I can add comments about this to the test case.